### PR TITLE
Generate a few ext/standard method synopses based on stubs - part 3

### PR DESCRIPTION
**Pull request by kocsismate**

### CODE REVIEW

File: reference/stream/functions/stream-socket-client.xml [hunk 1]
```diff
@@ -9,21 +9,21 @@
...
    Initiates a stream or datagram connection to the destination specified
-   by <parameter>remote_socket</parameter>.  The type of socket created
+   by <parameter>address</parameter>.  The type of socket created
    is determined by the transport specified using standard URL formatting:
    <literal>transport://target</literal>.  For Internet Domain sockets
    (AF_INET) such as TCP and UDP, the <literal>target</literal> portion
-   of the <parameter>remote_socket</parameter> parameter should consist of
+   of the <parameter>address</parameter> parameter should consist of
    a hostname or IP address followed by a colon and a port number.  For Unix
    domain sockets, the <parameter>target</parameter> portion should point
```
Comment (saundefined) [inline reference/stream/functions/stream-socket-client.xml:28]:
```suggestion
   domain sockets, the <literal>target</literal> portion should point
```
there is no `target` parameter, seems it should be a `literal` tag

**Review by saundefined — COMMENTED** [review]
Thanks! Looks good to me, except for one old nit.

File: reference/stream/functions/stream-socket-client.xml (hunk 1 above)
Comment (kocsismate) [reply reference/stream/functions/stream-socket-client.xml]: Thanks for the catch!

**Comment by cmb69** [comment]
Thank you!

**Pull request merged**